packages/Card/src/card.scss

```diff
@@ -20,13 +25,35 @@
   --ids-card-padding-xsmall: 0;
 }
 
+[data-brand="workleap"] {
+  /* Default */
+  --ids-card-font-family: var(--hop-body-md-font-family);
+  --ids-card-font-size: var(--hop-body-md-font-size);
+  --ids-card-background: var(--hop-neutral-surface);
+  --ids-card-border: 0.0625rem solid var(--hop-neutral-border-weak);
+  --ids-card-padding-mobile: var(--hop-space-inset-md);
+  --ids-card-padding: var(--hop-space-inset-xl);
+  --ids-card-shadow: none;
+  --ids-card-radius: var(--hop-shape-rounded-md);
+
+  /* Medium */
+  --ids-card-padding-medium: var(--hop-space-inset-lg);
+
+  /* Small */
+  --ids-card-padding-small: var(--hop-space-inset-md);
+
+  /* xSmall */
+  --ids-card-padding-xsmall: 0;
+}
+
 .ids-card {
   font-family: var(--ids-card-font-family);
-  padding: var(--ids-card-padding);
+  padding: var(--ids-card-padding-mobile);
```

Comment:
That's not bad here as there's only one but looking at modal/dialog moving all tokens at root and adding -mobile would create a lot of variables. Is that what we want?

Reply:
well I feel like all workleap tokens should be at the top so we can easily find them. It was either this or I call the other one padding-large and then keep the reset of --ids-card-padding in the media query and reset it to the large padding.

Reply:
Yes I understand. The in between state has some downsides.